libr/core/cbin.c

```diff
@@ -1867,13 +1873,17 @@ static int bin_symbols_internal(RCore *r, int mode, ut64 laddr, int va, ut64 at,
 		if (IS_MODE_RAD (mode)) {
 			r_cons_printf ("fs exports\n");
 		} else if (IS_MODE_NORMAL (mode)) {
-			r_cons_printf (printHere ? "" : "[Exports]\n");
+			r_cons_printf (printHere ? "" : "[Exports]\n"
```

Comment:
Even when `printHere` is true it still makes sense to have the "header". So, I'd just use another `r_cons_printf` to print the header in all cases.